src/kerberos/client/generators.rs

```diff
@@ -377,6 +377,7 @@ impl From<[u8; 24]> for ChecksumValues {
 }
 
 impl ChecksumValues {
+    #[allow(dead_code)]
     pub(crate) fn set_flags(&mut self, flags: GssFlags) {
         let flag_bits = flags.bits();
         let flag_bytes = flag_bits.to_le_bytes();
```

Comment:
nitpick: In general I prefer removing the dead code, but maybe this should be made public? I’m totally fine with removing this code too.

Reply:
This code became unused because I commented `flags` in the Kerberos implementation. I decided to keep it because it can be needed when we find out the cause of the CredSSP error

Reply:
Oh okay! Can you add a FIXME / TODO comment on the same line?

Reply:
sure. done